switch_topic_a: "/puma/brake/switch_a"
switch_topic_b: "/puma/brake/switch_b"

topic_brake_front: "/puma/brake/front_wheels"
topic_brake_rear: "/puma/brake/rear_wheels"

extra_steps_front: 200
extra_steps_rear: 200

pin_dir_front: 37
pin_step_front: 36
pin_dir_rear: 35
pin_step_rear: 33
